### DISCUSSION TITLE
UX Infra Focus on Resiliency

### DISCUSSION THREAD

**Discussion by gloriliale** [twilio-labs/paste#2370] [General] [answered]
## Focus on Resiliency

The UX Infrastructure team is currently strengthening the resiliency of our products. During this time we will be prioritizing resiliency work over requests for new components, patterns, contributions, or new features, unless that request is a direct blocker to your own resiliency work, such as migrating to Console Zen. 

When we've wrapped up our efforts, we will let you know!

As always, we welcome your feedback and questions on our existing products, and are excited about providing the systems and infrastructure to build resilient interfaces and experiences. Thanks for being a part of our community!

**Comment by loreina** [comment, marked as answer]
Commenting to mark this as answered
